.github/workflows/schedule-jira-sync.yml

```diff
@@ -0,0 +1,36 @@
+name: Sync Security Hub findings and Jira issues
+
+on:
+  workflow_dispatch: # for testing and manual runs
+  schedule:
+    - cron: '25 1 * * *' # daily at 6:25 am EST
+
+permissions:
+  id-token: write
+
+jobs:
+  sync:
+    name: Run sync
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+          role-to-assume: ${{ secrets.PRODUCTION_AWS_OIDC_ROLE_TO_ASSUME }}
+
+      - name: Sync Security Hub and Jira
+        uses: Enterprise-CMCS/security-hub-visibility@v1.0.1
+        with:
+          jira-token: ${{ secrets.JIRA_SERVICE_USER_TOKEN }}
+          jira-username: ${{ secrets.JIRA_SERVICE_USERNAME }}
+          jira-host: qmacbis.atlassian.net
+          jira-project-key: MDCT 
+          jira-epic-key: MDCT-2280 
+          jira-ignore-statuses: Done
```

Comment:
Question: We also use "Canceled" and "Closed". Should we add those?

Reply:
yes comma-separated list of Jira issue statuses that will be ignored when syncing can be added

Reply:
@samzsam Great! Will you add those so we have Done, Canceled, and Closed for all the PRs?